docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -36,6 +36,17 @@ In virtual environments, high CPU steal indicates that the virtual machines migh
 
 *Further Reading*: https://docs.couchbase.com/server/current/install/best-practices-vm.html[Virtualisation Best Practices]
 
+[#CB90043]
+=== Disk Commit Fail (CB90043)
+
+*Background*: A disk commit failure is when Couchbase Server cannot write data from memory to the file system.
+
+*Condition*: An item failed to be committed from memory to disk.
+
+*Remediation*: Review memcached.log to identify the cause, or contact Couchbase Technical Support.
```

Comment:
Can we have something more useful here? For example, "check your infrastructure" or something along those lines?